Podfile

```diff
@@ -1,8 +1,11 @@
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/foreseecode/podspecs'
+
```

Comment:
We don't want to merge this to master, right?

Reply:
Right. We'll need to do a final update to point back to the release pod and then hold this ticket waiting for the release.